### ISSUE TITLE
pressing the edit button shouldnt bring you to the bottom of the page

### ISSUE THREAD

**Issue by RustoMCSpit** [quillpad/quillpad#287]

**Comment by stardomains3** [comment]
I agree there should be an option to not scroll/not have keyboard pop up. Not to be ungrateful but it makes the app almost unusable to me as I have long notes. I'd prefer scrolling on my own and having keyboard pop when I tap where I want to edit.

**Comment by Solar-Simon** [comment]
I'm glad this issue has been raised. I'm experiencing the same problem — the scroll behavior breaks my workflow. Presumably a scroll to the cursor position is taking place? My expected behavior would be: When clicking "Edit", place the cursor in the first visible line.
